code/modules/mob/living/simple_animal/abnormality/he/der_freischutz.dm

```diff
@@ -14,12 +14,83 @@
 	work_damage_amount = 8 // This was halved what it should be.
 	work_damage_type = BLACK_DAMAGE
 
+	pinkable = FALSE
+
 	ego_list = list(
 		/datum/ego_datum/weapon/magicbullet,
 		/datum/ego_datum/weapon/magicpistol,
 		/datum/ego_datum/armor/magicbullet
 		)
 	gift_type =  /datum/ego_gifts/magicbullet
+	var/pink_cooldown = 30 SECONDS
+	var/pink_shots = 0
+	var/pink_declared = FALSE
+	COOLDOWN_DECLARE(pink)
+
+/mob/living/simple_animal/hostile/abnormality/der_freischutz/Life()
+	. = ..()
+	var/pink_midnight = FALSE
+	for(var/mob/living/simple_animal/hostile/A in GLOB.mob_list)
+		if(A != src)
+			if("pink_midnight" in A.faction)
+				pink_midnight = TRUE
+				break
+	if(pink_midnight)
+		if(!pink_declared)
+			addtimer(CALLBACK(GLOBAL_PROC, .proc/show_global_blurb, 5 SECONDS, "And so he once again picked up his gun for the one he loved.", 25))
```

Comment:
Again, the global text blurbs are meant to be *only* used by the big "oh shit" events. Right now that only includes Apocalypse Bird.

Reply:
Apocalypse Bird and General Bee at the moment. I feel like it's an appropriate way to alert people that something is going on but I could change to a chat alert if you'd like